tg_bot/modules/users.py

```diff
@@ -65,6 +69,133 @@ def broadcast(bot: Bot, update: Update):
                                             "due to being kicked.".format(failed))
 
 
+@run_async
+def restrict_group(bot: Bot, update: Update, args: List[str]) -> str:
+    message = update.effective_message  # type: Optional[Message]
+    chat_id_correct = False
+
+    # Check if there is only one argument
+    if not len(args) == 1:
+        message.reply_text("Incorrect number of arguments. Please use `/restrict chat_id`.",
+                           parse_mode=ParseMode.MARKDOWN)
```

Comment:
You need to return after this, or youll get a possible out of bounds exception thrown at the next line.